### ISSUE TITLE
bug: Storage Permission Issue for @capacitor/storage on Android 12

### ISSUE THREAD

**Issue by sattasundar** [ionic-team/capacitor#5790]
### Capacitor Version
    "@capacitor/android": "3.6.0",
    "@capacitor/app": "1.1.1",
    "@capacitor/camera": "^1.3.1",
    "@capacitor/core": "3.6.0",
    "@capacitor/storage": "^1.2.4",
    "@ionic/angular": "6.1.15",

### AndroidManifest.xml
```
    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

### Platform(s)
Android 12

### Current Behavior
While saving to Storage, it's not working. I checked in App info in  device settings, Storage permission was not enabled. 

### Expected Behavior
Should enable storage permission or else there should be methods checkPermissions() / requestPermissions(…) like @capacitor/camera plugin.

**Comment by jcesarmobile** [comment]
Despite the plugin name, it's not really storing the values to the filesystem, it uses [Android SharedPreferences ](https://developer.android.com/reference/android/content/SharedPreferences), which requires no permissions.
In fact we have renamed the plugin to capacitor/preferences to avoid that confusion. 
I've tested and it's working fine on my Android 12 device, make sure you are storing strings only (as key and as value).
